# Remove unnecessary temp file verification that causes "no free devices"

--- vmx_fixed2.c.orig
+++ vmx_fixed2.c
@@ -706,11 +706,6 @@
 		close(fd);
 		free(bootimg);
 		
-		/* Verify the file exists and is readable */
-		fd = open(tmpfile, OREAD);
-		if(fd < 0)
-			sysfatal("cannot open boot image file %s for reading: %r", tmpfile);
-		close(fd);
-		
 		/* VMX will load this as kernel */
 		argv[0] = tmpfile;